src/renderer/components/utils/Slider.tsx

```diff
@@ -124,13 +124,12 @@ export default class Slider extends React.Component<Props, State> {
     private moveInView(elementId: number) {
         const max = -this.contentRef.current.offsetWidth + this.wrapperRef.current.offsetWidth;
         const element = this.contentElRefs[elementId];
-        if (element) {
-            let elementPosition = -element.offsetLeft;
-            const isVisible = this.isElementVisible(elementId);
-            if (!isVisible) {
-                elementPosition = elementPosition > 0 ? 0 : elementPosition < max ? max : elementPosition;
-                this.setState({position: elementPosition, refreshVisible: true});
-            }
+
+        let elementPosition = -element.offsetLeft;
+        const isVisible = this.isElementVisible(elementId);
+        if (!isVisible) {
+            elementPosition = elementPosition > 0 ? 0 : elementPosition < max ? max : elementPosition;
+            this.setState({position: elementPosition, refreshVisible: true});
```

Comment:
Hello @manuellagrand could you please file a new issue describing how to reproduce the bug you identified when you introduced this check for undefined/null/falsy `element`. Thank you!